win32ss/user/ntuser/callback.c

```diff
@@ -35,6 +35,7 @@ IntCbAllocateMemory(ULONG Size)
       return NULL;
    }
 
+   RtlZeroMemory(Mem, Size + sizeof(INT_CALLBACK_HEADER));
```

Comment:
This is probably not really necessary for all callers. Let's find the ones that require it instead and do it in the caller?

Reply:
Yes, I will find all of them.